modules/olm-mirroring-catalog.adoc

```diff
@@ -0,0 +1,300 @@
+// Module included in the following assemblies:
+//
+// * operators/olm-restricted-networks.adoc
+// * operators/olm-managing-custom-catalogs.adoc
+// * migration/migrating_3_4/deploying-cam-3-4.adoc
+// * migration/migrating_4_1_4/deploying-cam-4-1-4.adoc
+// * migration/migrating_4_2_4/deploying-cam-4-2-4.adoc
+
+[id="olm-mirror-catalog_{context}"]
+= Mirroring an Operator catalog
+
+You can mirror the Operator content of a Red Hat-provided catalog, or a custom
+catalog, into a container image registry using the `oc adm catalog mirror`
+command. The target registry must support
+link:https://docs.docker.com/registry/spec/manifest-v2-2/[Docker v2-2]. For a
+cluster on a restricted network, this registry can be a registry that the
+cluster has network access to, such as a mirror registry created during a
+restricted network cluster installation.
+
+[IMPORTANT]
+====
+The internal registry of your {product-title} cluster cannot be used as the
+target registry because it does not support pushing without a tag, which is
+required during the mirroring process.
+====
+
+You must also mirror the Red Hat-provided index image from Quay.io, or push your
```

Comment:
This PR is relevant (though not yet merged): https://github.com/openshift/oc/pull/611

when this merges, `oc adm catalog mirror` will mirror the index image as well. (it will also support airgap installs where you mirror from redhat -> file and then later from file -> airgapped registry)

Reply:
:nice: